src/parser.ts

```diff
@@ -409,7 +409,8 @@ function parseIncludeInputs (ctx: any): {inputValue: any; inputType: InputType}
 
 function getInputValue (ctx: any) {
     const {inputs, interpolationKey, configFilePath, inputsSpecification} = ctx;
-    const inputValue = inputs[interpolationKey] || inputsSpecification.spec.inputs[interpolationKey]?.default;
+    const inputValue = inputs[interpolationKey]
+      ?? inputsSpecification.spec.inputs[interpolationKey]?.default;
```

Comment:
```suggestion
    const inputValue = inputs[interpolationKey] ?? inputsSpecification.spec.inputs[interpolationKey]?.default;
```

Reply:
i see you prefer this syntax, 

so added https://github.com/firecow/gitlab-ci-local/pull/1532/commits/116267b4123c7ce48cd6c4084a4d64251e707d45 

to enforce it

Reply:
I :heart: you :smile: